vue.config.js

```diff
@@ -1,4 +1,15 @@
 module.exports = {
   filenameHashing: false,
   productionSourceMap: false,
+  devServer: {
+    proxy: {
+      '/proxy': {
+        target: 'https://lobste.rs/',
+        changeOrigin: true,
+        pathRewrite: {
+          '^/proxy': '',
+        },
+      },
+    },
+  },
```

Comment:
There's no need to implement such a proxy since Lobste.rs seems to be giving the data simply at the `/hottest.json` path, each client can request there on their own, essentially removing the need for supporting a backend for this feature. 

I assume you had CORS issues locally because you probably used localhost to view your modifications instead of loading unpacked extension, which would have CORS limitations. I haven't tested yet, but you should be able to add Lobster's domain to the `manifest.json` [like explained here](https://developer.chrome.com/extensions/xhr), and it should work.

Reply:
Ah I'm new to chrome extensions, I'm used to the browser's blockage. 
Thanks for the explanation.